api-conversion/README.md

```diff
@@ -0,0 +1,126 @@
+# API Conversion CLI Tool
+
+With Strimzi's CRD evolving, there needs to be a way to easily migrate existing CR configuration to the new CRD schema.
+That is why we introduced the `v1beta2` version of the schema which removes several deprecated fields.
+To make it as easy as possible to convert existing Strimzi custom resources, a conversion tool is provided.
+
+The tool can operate in two modes:
+* Converting YAML files
+* Converting Kubernetes resources
+
+You can list the available features using the `help` command:
+
+```
+> bin/api-conversion.sh help
+Usage: bin/api-conversion.sh [-hV] [COMMAND]
+Conversion tool for Strimzi Custom Resources
+  -h, --help      Show this help message and exit.
+  -V, --version   Print version information and exit.
```

Comment:
```suggestion
  -h, --help      Show this help message and exit
  -V, --version   Print version information and exit
```

Reply:
This is generated by the PicoCLI library. I can delete it from the README, but not from the actual output. So I would leave it as it is.